css/components/alert.css

```diff
@@ -0,0 +1,44 @@
+.alert {
+  border-radius: 0.3rem;
```

Comment:
Don't use REM for border-radius as they'll become shape-shifters based on pixel density.
Rest looks good!

Reply:
Done!